inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/KBUtility.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.tudarmstadt.ukp.inception.search.index.mtas;
+
+import java.util.Optional;
+
+import org.eclipse.rdf4j.model.Resource;
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.model.ValueFactory;
+import org.eclipse.rdf4j.query.QueryEvaluationException;
+import org.eclipse.rdf4j.repository.RepositoryConnection;
+import org.eclipse.rdf4j.repository.RepositoryResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.annotation.feature.FeatureSupport;
+import de.tudarmstadt.ukp.clarin.webanno.api.annotation.feature.FeatureSupportRegistry;
+import de.tudarmstadt.ukp.clarin.webanno.model.AnnotationFeature;
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.inception.kb.ConceptFeatureTraits;
+import de.tudarmstadt.ukp.inception.kb.KnowledgeBaseService;
+import de.tudarmstadt.ukp.inception.kb.graph.KBConcept;
+import de.tudarmstadt.ukp.inception.kb.graph.KBHandle;
+import de.tudarmstadt.ukp.inception.kb.graph.KBInstance;
+import de.tudarmstadt.ukp.inception.kb.graph.KBObject;
+import de.tudarmstadt.ukp.inception.kb.graph.RdfUtils;
+import de.tudarmstadt.ukp.inception.kb.model.KnowledgeBase;
+
+public class KBUtility {
+    private final static Logger log = LoggerFactory.getLogger(KBUtility.class.getName());
+    /**
+     * Returns class type of {@link KBObject}
+     * 
+     * @param kbObject
+     * @return {@link Optional<String>} 
+     */
+    public Optional<String> getType(Optional<KBObject> kbObject) {
+        return kbObject.map((p) -> p.getClass().getSimpleName());
+    }
+    /**
+     * Read identifier IRI and return {@link KBHandle}
+     * @param {@link FeatureSupportRegistry} featureSupportRegistry
```

Comment:
Either properly document the parameters/return values or remove them. Type and parameter name are already auto-generated by JAvaDoc anyway - repeating them does not contribute anything. This applies to all JavaDocs.